src/metabase/search/impl.clj

```diff
@@ -228,13 +228,13 @@
 
 ;; This forwarding is here for tests, we should clean those up.
 
-(defmethod search.api/results :default [search-ctx]
+(defmethod search.api/results nil [search-ctx]
```

Comment:
why?

Reply:
My thinking was to avoid endless recursion if we haven't implemented the default case.

That said, this also has a rough edge - hard failure if we deprecate an engine.

I'll think I'll rework it to still be a default dispatch, but to short circuit if its already set to default 🤔